test/collection_test.js

```diff
@@ -157,6 +159,45 @@ describe("Collection", () => {
       });
     });
 
+    describe("hooks registration", () => {
+      function registerHooks(hooks) {
+        return new Collection(TEST_BUCKET_NAME, TEST_COLLECTION_NAME, api, {
+          hooks,
+          adapter: IDB
```

Comment:
This is how it's done in the tests at other locations, I just copied it over. Could be addressed as a separate issue I guess.